signals/main.py

```diff
@@ -47,6 +49,22 @@ def project_specified(ctx, param, value):
         ctx.exit()
 
 
+# Schema
+def validate_schema_path(ctx, param, value):
+    if value.startswith('~'):
+        value = os.path.expanduser(value)
+    elif value.startswith('.'):
+        value = os.path.abspath(value)
+
+    exists = os.path.isfile(value)
+
+    if not exists:
```

Comment:
Might just want to inline the exists check from above since we don't really need it to be a variable